samples/excel/30-events/events-comment-event-handler.yaml

```diff
@@ -0,0 +1,195 @@
+order: 13
+id: excel-events-comment-event-handler
```

Comment:
This is outside of the scope of this PR, but we should reconsider if an "events" category makes sense. It might be a better organizational tactic to put the comment events snippet with the other comment samples (same for charts, tables, and the rest). Again, that's beyond this PR, but something to think about.

Reply:
I had the same thought while selecting the appropriate folder for this new sample. I put the sample in this folder because it's the most logical location while the 'events' category exists, but don't think an 'events' category is particularly helpful, and I do want to reorganize the samples in this folder in the future. Thanks for mentioning this!